frontend/src/modules/labwork/actions.ts

```diff
@@ -5,17 +5,17 @@ import { refreshSamplesAtStep } from '../labworkSteps/actions'
 import { findChangedStepsInSummary, processFMSLabworkSummary } from './services'
 
 export const GET_LABWORK_SUMMARY = createNetworkActionTypes('SAMPLE-NEXT-STEP.GET_LABWORK_SUMMARY')
+export const LIST_TABLE = createNetworkActionTypes("SAMPLE-NEXT-STEP.LIST_TABLE");
```

Comment:
I'm not sure why you added this action definition - it doesn't seem to be used anywhere.